sympy/functions/elementary/trigonometric.py

```diff
@@ -108,6 +109,24 @@ def _period(self, general_period, symbol=None):
         raise NotImplementedError("Use the periodicity function instead.")
 
 
+@cacheit
+def _table2():
```

Comment:
Why not just make this a global variable like:
```
_TABLE2 = {12: (3, 4), ...}
```

Reply:
Actually I suppose having it as a function means that there's less import time overhead and reduced memory if it never actually gets used.